mantidproject/mantid
# Replaced ICat Credentials in Unit Tests

Fixes #10506 

This code replaced the credentials used for logging into ICat when running unit tests. It was then found that test were taking ~55 seconds each to run and so tests were refactored to improve speed.

To Test:
- Run the ICat unit tests
- Confirm that the tests are successfully logging into ICat (via the test output)
- Confirm that the tests take a reasonable amount of time
